packages/core/package.json

```diff
@@ -65,19 +62,5 @@
     "nodemon": "2.0.12",
     "esbuild-loader": "2.13.1"
   },
-  "devDependencies": {
-    "@types/babel-core": "6.25.7",
-    "@types/babel-plugin-tester": "9.0.4",
-    "babel-plugin-tester": "10.1.0"
-  },
-  "gitHead": "8be6a35c2dfd5aaeb12d55be4f0c77eefceb7762",
-  "scripts": {
```

Comment:
Confused about how/why these scripts are being removed... are there next steps planned here?

Reply:
They've been moved to `internal`?